CHANGELOG.md

```diff
@@ -11,3 +11,5 @@
 * Add BinaryWriter as new entry point for NodeWriter, EdgeWriter and alias writers.
 
 * Meta.json files are no longer needed by the converter. Remove meta path argument from MultiWorkerConverter and Dispatchers.
+
+* Support nodes and their outgoing edges on different partitions.
```

Comment:
What does this mean? I don't see any changes to edges, only node features were updated.

Reply:
When an edge is added to a partition without its source node added first, this PR will add a dummy node with type = -1 - this way we support having a nodes outgoing edges on different partitions.

Reply:
That's not how does [the decoder interface work right now](https://github.com/microsoft/DeepGNN/blob/main/src/python/deepgnn/graph_engine/snark/decoders.py#L36) :

> this is a generator that yields a node then its outgoing edges in order.

This means all edges will end up in the same partition. It seems like you are missing changes in the converter interface for this PR.

Reply:
This is a documentation issue, I will fix.